fe2-android/app/build.gradle

```diff
@@ -16,179 +16,147 @@ android {
 
         testInstrumentationRunner "com.github.dedis.popstellar.HiltTestRunner"
 
+        // The following feature is not working properly with the coverage on the current build version
```

Comment:
which following feature?

Reply:
I think this will be clearer with the changes below.
I would also reference https://issuetracker.google.com/issues/123987001 explicitly in the comment, so that we can easily check on the status of the resolution.

They seem to be indicating a sort of temporary fix there, by the way, is it worth looking into?